examples/draft-0-10-0/playground/src/DraftJsRichEditorExample.js

```diff
@@ -0,0 +1,212 @@
+/**
```

Comment:
This was extracted from the rich.html example, with minor modification to use the onChange from the wrapper parent